tests/fiaas_deploy_daemon/deployer/kubernetes/test_autoscaler.py

```diff
@@ -31,20 +31,20 @@ def test_default_spec_should_create_no_autoscaler(app_spec):
 
 
 def test_autoscaler_enabled_and_1_replica_gives_no_autoscaler(app_spec):
-    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=2, cpu_threshold_percentage=50))
+    app_spec = app_spec._replace(autoscaler=AutoscalerSpec(enabled=True, min_replicas=1, max_replicas=2, cpu_threshold_percentage=50))
```

Comment:
This makes the test confusing; it should be verifying that if min=max=1 then no autoscaler, but if they're different, then there _should_ be an autoscaler